src/components/dls/Skeleton/Skeleton.module.scss

```diff
@@ -23,18 +14,16 @@ $animationDuration: 8s;
 
 .active::before {
   border-radius: var(--border-radius-default);
-  background-image: linear-gradient(
-    270deg,
-    var(--shade-0),
-    var(--shade-1),
-    var(--shade-2),
-    var(--shade-1),
-    var(--shade-0)
-  );
-  background-size: 400% 100%; // increase background size's width to 4x to make background animate-able
-  animation: loading $animationDuration ease-in-out infinite;
-
-  content: "";
+
+  [data-theme="dark"] & {
+    background-color: rgba(255, 255, 255, 0.13);
+  }
+  [data-theme="light"] & {
+    background-color: rgba(0, 0, 0, 0.11);
+  }
+  animation: 1.5s ease-in-out 0.5s infinite normal none running loading;
+
```

Comment:
Can you post a before/after for this change? Since it touches our dls component.

Reply:
**before:**

https://user-images.githubusercontent.com/3989750/141662811-b4290f5f-b3f9-41b2-bf3a-b71e63916321.mov


https://user-images.githubusercontent.com/3989750/141662812-56a57b84-d9f4-4f70-abe4-6dc22cbcd9a3.mov

**after:**

https://user-images.githubusercontent.com/3989750/141662825-32ccdab6-a7be-4397-8d0b-104b2ccbd03f.mov


https://user-images.githubusercontent.com/3989750/141662826-227c174b-131c-4a29-93a7-84f8bcba63ff.mov